src/components/add-form/styles.css

```diff
@@ -36,22 +36,14 @@
   font-weight: 500;
 }
 
-.time-frame-label {
-  margin-bottom: 8px;
-}
-
-.item-name {
-  padding: 10px 0;
-}
-
 .time-frame-legend {
   text-align: center;
   font-size: 16px;
   font-weight: 700;
 }
 
-.time-frame-radio {
-  width: 30px;
+.time-frame-radio input {
```

Comment:
Here, instead of giving the tag selector, if the input had the className given it would've worked too ^^